bhagavad_gita/lib/Constant/quotes.dart

```diff
@@ -104,4 +104,125 @@ var quotesList = [
 
 var quotesListHindi = [
   "जब भी धर्म का ह्रास होता है और जीवन का उद्देश्य भूल जाता है, तो मैं स्वयं को पृथ्वी पर प्रकट करता हूं। मैं हर युग में अच्छाई की रक्षा, बुराई को नष्ट करने और धर्म की स्थापना के लिए पैदा हुआ हूं।",
```

Comment:
Please remove this quote since it was translated from English using Google translate and has lots of errors.